java/show-labels-on-layer/README.md

```diff
@@ -1,26 +1,43 @@
-# Show Labels on Layer
-Demonstrates how to show labels on a feature layer.
+# Show labels on layer
 
-![Show Labels on Layer App](show-labels-on-layer.png)
+Display custom labels on a feature layer.
+
+![Image of show labels on layers](show-labels-on-layer.png)
+
+## Use case
+
+Labeling features is useful to visually display a key piece of information or attribute of a feature on a map. For example, you may want to label rivers or streets with their names. 
 
 ## How to use the sample
-Simply load the sample. Notice the way labels change with different zoom levels.
+
+Pan and zoom around the United States. Labels for congressional districts will be shown in red for Republican districts and blue for Democrat districts. Notice how labels pop into view as you zoom in.
```

Comment:
Just a note for the future: this sample starts zoomed all the way out to the full world extent and you have to zoom in really close to a city to actually see anything.

Reply:
hmm, this sounds like a service issue. the labels you're seeing belong the basemap. it can take a while for the results to display (sometimes it's really quick, other times 30 secs or so). When it works you should see blue and red areas and the labels maintain their size regardless of scale.